src/main/java/io/cryostat/net/web/http/api/v1/ProbeTemplateDeleteHandler.java

```diff
@@ -0,0 +1,125 @@
+ * (b) any piece of software and/or hardware listed in the lrgrwrks.txt file if
+ * one is included with the Software (each a "Larger Work" to which the Software
+ * is contributed by such licensors),
+ *
+ * without restriction, including without limitation the rights to copy, create
+ * derivative works of, display, perform, and distribute the Software and make,
+ * use, sell, offer for sale, import, export, have made, and have sold the
+ * Software and the Larger Work(s), and to sublicense the foregoing rights on
+ * either these or other terms.
+ *
+ * This license is subject to the following condition:
+ * The above copyright notice and either this complete permission notice or at
+ * a minimum a reference to the UPL must be included in all copies or
+ * substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package io.cryostat.net.web.http.api.v1;
+
+import io.cryostat.core.agent.LocalProbeTemplateService;
+import io.cryostat.core.log.Logger;
+import io.cryostat.core.sys.FileSystem;
+import io.cryostat.messaging.notifications.NotificationFactory;
+import io.cryostat.net.AuthManager;
+import io.cryostat.net.security.ResourceAction;
+import io.cryostat.net.web.http.AbstractAuthenticatedRequestHandler;
+import io.cryostat.net.web.http.HttpMimeType;
+import io.cryostat.net.web.http.api.ApiVersion;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.ext.web.handler.impl.HttpStatusException;
+
+import javax.inject.Inject;
+import java.util.EnumSet;
+import java.util.Map;
+import java.util.Set;
+
+public class ProbeTemplateDeleteHandler extends AbstractAuthenticatedRequestHandler {
+
+
+    static final String PATH = "probes";
+
+    private static Logger logger;
+    private final NotificationFactory notificationFactory;
+    private final LocalProbeTemplateService probeTemplateService;
+    private final FileSystem fs;
+    private static final String  NOTIFICATION_CATEGORY = "ProbeTemplateUploaded";
+
+    @Inject
+    ProbeTemplateDeleteHandler(
+        AuthManager auth,
+        NotificationFactory notificationFactory,
+        LocalProbeTemplateService probeTemplateService,
+        Logger logger,
+        FileSystem fs
+    ) {
+        super(auth);
+        this.notificationFactory = notificationFactory;
+        this.logger = logger;
+        this.probeTemplateService = probeTemplateService;
+        this.fs = fs;
+    }
+
+    @Override
+    public ApiVersion apiVersion() {
+        return ApiVersion.V1;
```

Comment:
This should be a v2 handler

Reply:
Actually, it should be a beta API handler, but the implementation should extend `AbstractV2RequestHandler`